### PR TITLE
count the chunks and export that

### PR DESCRIPTION
**Pull request by bgriffinte**

### CODE REVIEW

**Comment by bgriffinte** [comment]
Not in `sharding_topology.go` because making this a command line option strongly suggests putting it in a new file. So I did that.

File: mongodb_exporter.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ var (
 	collectTopF                  = kingpin.Flag("collect.topmetrics", "Enable collection of table top metrics").Bool()
 	collectIndexUsageF           = kingpin.Flag("collect.indexusage", "Enable collection of per index usage stats").Bool()
 	mongodbCollectConnPoolStatsF = kingpin.Flag("collect.connpoolstats", "Collect MongoDB connpoolstats").Bool()
+	collectChunksF               = kingpin.Flag("collect.chunkcounts", "Enable counting of chunks per shard (possibly slow)").Bool()
```
Comment (shashank-te) [inline mongodb_exporter.go]: desc - per shard per collection
Instead of just slow, we could say that it could be slow on clusters with millions of chunks.

Comment (shashank-te) [reply mongodb_exporter.go]: we could call it `collect.chunkstats` to leave room for more specific statistics than just count.

**Comment by shashank-te** [comment]
@bgriffinte is this ready for review?

**Pull request merged**